Makefile

```diff
@@ -61,6 +61,11 @@ check-ink-wrapper:
 	cd ink-wrapper && cargo fmt --all --check
 	cd ink-wrapper && cargo clippy --all-features -- --no-deps -D warnings
 
+.PHONY: check-ink-wrapper-types
+check-ink-wrapper-types:
+	cd ink-wrapper-types && cargo fmt --all --check
+	cd ink-wrapper-types && cargo clippy --all-features -- --no-deps -D warnings
```

Comment:
Do you really need to `cd` twice?

Reply:
Yah, it's not bash, it's `make`. There are a couple other solutions, like running make in a subdirectory, but then you have to split you `Makefile` into parts that live in those directories. Or:

```
cd ink-wrapper-types && cargo fmt --all --check && cargo clippy --all-features -- --no-deps -D warnings
```

but I like it less from an esthetic standpoint.